23-07-workshop/finished/Leavitts_Footsteps_solutions.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<p>the line label didn't print on this (and we should label the axes, as well)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>